portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,207 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Katherine's Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/3/w3.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
     <script src="script.js"></script>
   </head>
+
+  
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+
+    <!-- Navigation -->
+    <nav class="topnav" id="myTopnav">
+        <a href="#home" class="w3-button w3-bar-item">Home</a>
+        <a href="#resume" class="w3-button w3-bar-item">Resume</a>
+        <a href="#projects" class="w3-button w3-bar-item">Projects</a>
+        <a href="#contact" class="w3-button w3-bar-item">Contact</a>
+        <a href="javascript:void(0);" class="icon" onclick="navFunction()">
+            <i class="fa fa-bars"></i>
+        </a>
+    </nav>
+
+    <!-- Slide Show -->
+    <section>
+        <img class="mySlides" src="images/zion-national-park.jpeg"
```

Comment:
Please use hyphen instead of camel case in html/css https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters

Same applies elsewhere

e.g, `my-slides`

Reply:
okay I have renamed my classes and other identifiers, thank you